src/plugins/discover/public/utils/get_sharing_data.ts

```diff
@@ -59,11 +59,21 @@ export async function getSharingData(
     }
   }
 
+  const absoluteTimeFilter = data.query.timefilter.timefilter.createFilter(index);
+  const relativeTimeFilter = data.query.timefilter.timefilter.createRelativeFilter(index);
   return {
     getSearchSource: (absoluteTime?: boolean): SerializedSearchSourceFields => {
-      const timeFilter = absoluteTime
-        ? data.query.timefilter.timefilter.createFilter(index)
-        : data.query.timefilter.timefilter.createRelativeFilter(index);
+      const absoluteFilter = data.query.timefilter.timefilter.createFilter(index);
```

Comment:
@Dmitriynj this should be redundant , right, since you already could use `absoluteTimeFilter` of the upper scope